src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import classNames from "classnames";
+export const Tabs = ({tabs, activeTabId, onTabSelected, setOnTabSelected, setActiveTabId}) => {
+    let i = 0;
+    return (
+        <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            
+            return (
+                <li className={classNames({'is-active' : activeTabId === tab.id})} 
+                data-cy="Tab"
+                onClick={ () => {
+                    setOnTabSelected(tab)
+                    setActiveTabId(tab.id)
+                }}
+                key={tab.id}
+                >
+                    <a href={`#${activeTabId.id}`} data-cy="TabLink">
```

Comment:
The `href` attribute should reference the tab's ID, not `activeTabId.id`. It should be `href={`#${tab.id}`}` to ensure each tab link corresponds to the correct tab.